ui/vite.config.ts

```diff
@@ -3,6 +3,7 @@ import react from "@vitejs/plugin-react";
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  base: "./",
```

Comment:
this forces us to always need to start from the home page: we cannot directly go to the traces pages.